perceval/backends/core/confluence.py

```diff
@@ -332,6 +352,8 @@ def contents(self, from_date=DEFAULT_DATETIME,
         :param limit: maximum number of contents to fetch per request
         """
         resource = self.RCONTENTS + '/' + self.MSEARCH
+        if self.add_ancestors:
+            resource = resource + "?expand=ancestors"
```

Comment:
instead of hard-coding `?expand=ancestors`, you could leverage on variables and reference them in the code (as similarly done in other cases such as `RCONTENTS`, `PSTATUS` and `PVERSION`)